action.Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM node:10-stretch AS builder
+LABEL maintainer="datapunt@amsterdam.nl"
+
+WORKDIR /app
+
+# Run updates and cleanup
+RUN apt-get update && apt-get install -y \
+      netcat \
+      git && \
+      rm -rf /var/lib/apt/lists/*
+
+#  Changing git URL because network is blocking git protocol...
+RUN git config --global url."https://".insteadOf git://
+RUN git config --global url."https://github.com/".insteadOf git@github.com:
+
+COPY internals /app/internals
+COPY server /app/server
+COPY .gitignore \
```

Comment:
Why are we copying all these git files?
Doesn't seem like git is used in this container.